u8timeseries/utils/mising_values.py

```diff
@@ -0,0 +1,121 @@
+from u8timeseries.timeseries import TimeSeries
+import matplotlib.pyplot as plt
+import numpy as np
+from typing import Tuple, List
+
+
+def na_ratio(ts: 'TimeSeries') -> float:
+    """
+    Computes the ratio of missing values in percent.
+
+    :param ts: The TimeSeries to check for missing values.
+    :return: A float, the percentage of missing values in `ts`.
+    """
+
+    return 100 * ts.pd_series().isnull().sum() / len(ts)
+
+
+def fillna_0(ts: 'TimeSeries') -> 'TimeSeries':
+    """
+    Fills the missing values of `ts` with zeroes only.
+
+    :param ts: The TimeSeries to check for missing values.
+    :return: A TimeSeries, `ts` with al missing values set to 0.
+    """
+    return TimeSeries.from_times_and_values(ts.time_index(), ts.pd_series().fillna(value=0))
+
+
+def nan_structure_visual(ts: 'TimeSeries'):
+    """
+    Plots the indicator function of missing values of `ts`.
+
+    Missing values have value 1 and non-missing values are 0.
+
+    :param ts: The TimeSeries to check.
+    """
+
+    nans = np.isnan(ts.values())
+
+    plt.scatter(np.arange(len(nans)), nans)
+    plt.show()
+
+
+def change_of_state(ts: 'TimeSeries') -> Tuple[int, List]:
+    """
+    Determines the indices where `ts` changes from missing values to non-missing and vice-versa.
+
+    :param ts: The TimeSeries to analyze.
+    :return: A tuple (a, l), where a is the number of change of states and l is the list of indices where \
+    changes of states occur.
+    """
+
+    a = np.where(np.diff(np.isnan(ts.values())) is True)
+
+    return len(a[0]), list(a[0])
+
+
+def auto_fillna(ts: 'TimeSeries') -> 'TimeSeries':
+    """
+    This function automatically fills the missing value in the TimeSeries `ts`.
```

Comment:
Perhaps a comment here saying that it is assumed that missing values are already np.nan in the TimeSeries. This is a good assumption most of the time, but there might be other "missing value" symbols (sometimes even numbers such as 0 or -inf are used as missing values symbols)... users should be aware I think.